Formula/gst-plugins-bad.rb

```diff
@@ -55,6 +56,8 @@ def install
       system "./autogen.sh"
     end
 
+    args << "--with-gtk=3.0" if build.with? "gtk+3"
```

Comment:
Nit, but please shuffle this up so it's underneath the other `args <<` line rather than down here by itself.